web/src/app/administration/components/workbasket-access-items/workbasket-access-items.component.html

```diff
@@ -23,86 +34,101 @@
-							</button>
+							<input class="workbasket-access-items__permission-checkbox" type="checkbox"
+								aria-label="selectRow" aria-labelledby="selectRow" (change)="selectRow($event, index)">
 						</td>
 
 						<!-- ACCESS ID -->
 						<td *ngIf="(accessItemsCustomization$ | async)?.accessId.lookupField else accessIdInput"
-                class="workbasket-access-items__typeahead"
-                [ngClass]="{	'has-warning': (accessItemsClone[index].accessId !== accessItem.value.accessId),
+							class="workbasket-access-items__typeahead" [ngClass]="{	'has-warning': (accessItemsClone[index].accessId !== accessItem.value.accessId),
                 							'has-error': !accessItem.value.accessId }">
 
-							<taskana-shared-type-ahead formControlName="accessId" placeHolderMessage="* Access id is required"
-                                         [validationValue]="toggleValidationAccessIdMap.get(index)"
-                                         [displayError]="!isFieldValid('accessItem.value.accessId', index)"
-                                         (selectedItem)="accessItemSelected($event, index)"
-                                         (inputField)="focusNewInput($event)">
-              </taskana-shared-type-ahead>
-            </td>
+							<taskana-shared-type-ahead formControlName="accessId"
+								placeHolderMessage="* Access id is required"
+								[validationValue]="toggleValidationAccessIdMap.get(index)"
+								[displayError]="!isFieldValid('accessItem.value.accessId', index)"
+								(selectedItem)="accessItemSelected($event, index)">
+							</taskana-shared-type-ahead>
+						</td>
 
 						<ng-template #accessIdInput>
 							<td>
 								<div [ngClass]="{ 'has-warning': (accessItemsClone[index].accessId !== accessItem.value.accessId),
 																  'has-error': !accessItem.value.accessId && formsValidatorService.formSubmitAttempt}">
-                  <input matInput type="text" formControlName="accessId"
-                         placeholder="{{accessItem.invalid ? '* Access id is required': ''}}"
-                         [@validation]="toggleValidationAccessIdMap.get(index)" #htmlInputElement>
+									<input matInput type="text" formControlName="accessId"
+										placeholder="{{accessItem.invalid ? '* Access id is required': ''}}"
+										[@validation]="toggleValidationAccessIdMap.get(index)" #htmlInputElement>
 								</div>
 							</td>
 						</ng-template>
 
 						<!-- SELECT ALL -->
 						<td>
-                <input class="workbasket-access-items__permission-checkbox" type="checkbox"  id="checkbox-{{index}}-00" (change)="checkAll(index, $event)" aria-label="checkAll" aria-labelledby="checkAll">
-            </td>
+							<input class="workbasket-access-items__permission-checkbox" type="checkbox"
+								id="checkbox-{{index}}-00" (change)="checkAll(index, $event)" aria-label="checkAll"
+								aria-labelledby="checkAll">
+						</td>
```

Comment:
I think this ticket https://taskana.atlassian.net/browse/TSK-1044 should be part of the rework of the workbasket access items. We can also work on this ticket in a separate PR.